howtos/cross_building.rst

```diff
@@ -143,3 +143,155 @@ If you have your project building automated with conan you can then just call ``
 
 
 So, now you can commit your profile files to a repository and use them for cross-build your projects.
+
+
+
+.. _cross_building_android:
+
+
+Android
+-------
+
+
+Cross bulding a library for Android is very similar to the previous examples, except the complexity of managing different
+architectures (armeabi, armeabi-v7a, x86, arm64-v8a) and the Android api levels.
+
+Creating a toolchain for the specific settings can be a tedious task. The Android NDK offers a mechanism to create a
+`standalone toolchain <https://developer.android.com/ndk/guides/standalone_toolchain.html?hl=es>`_ for different settings.
+
+We created a recipe ``android-toolchain/r13b@lasote/testing`` to be used as a :ref:`build requirement<build_requires>`.
```

Comment:
Not sure if the "bare" example of the toolachain without the "build_requires" should be explained somewhere too.